app/controllers/pxe_controller.rb

```diff
@@ -265,8 +263,8 @@ def replace_right_cell(options = {})
       presenter.hide(:form_buttons_div)
     end
 
-    # disable toolbar and buttons for react add/edit pxe server form
-    if @in_a_form && nodetype == 'ps'
+    # disable toolbar and buttons for react add/edit pxe server and pxe customization template forms
+    if @in_a_form && nodetype.start_with?('ps', 'ct-')
```

Comment:
@h-kataria I think this form is also ready for review, but if https://github.com/ManageIQ/manageiq-ui-classic/pull/7595 is merged first this line will need to be edited and vice versa.